sql/mysqld.cc

```diff
@@ -7006,9 +7010,22 @@ static int show_heartbeat_period(THD *thd, SHOW_VAR *var, char *buff,
   return 0;
 }
 
-
 #endif /* HAVE_REPLICATION */
 
+static int show_max_used_connections_time(THD *thd, SHOW_VAR *var, char *buff,
+                                 enum enum_var_type scope)
+{
+
+  char time_str[20];
+  get_date(time_str, GETDATE_DATE_TIME, max_used_connections_time);
+
+  var->type= SHOW_CHAR;
+  var->value= buff;
+
+  sprintf(buff, "%s", time_str);
```

Comment:
I quess you want to use safer version `snprintf`  here like you did with 567b68129943a1cceab1d7b4c68e2a4ba011cdc0 and PR #2480 ?

Reply:
or better, why not just `get_date` straight into `buff` and avoid the copy.